service/orchestrator/metrics_test.go

```diff
@@ -47,20 +47,132 @@ var ErrSomeError = errors.New("some error")
 
 const MockMetricID = "SomeMetric"
 
+func TestService_loadMetrics(t *testing.T) {
```

Comment:
I like "TestService_...". Would be good to use this pattern consistent throughout the project.